### ISSUE TITLE
Chrome 105 on MacOS 12.2 (21D49) not displaying encrypted video over Screen Mirroring (AirPlay) using Video.JS 7.20.2, and the EME plugin 5.0.1

### ISSUE THREAD

**Issue by nick-myers-dt** [videojs/videojs-contrib-eme#174]
Chrome Version       : 105.0.5195.125
URLs (if applicable) : https://codepen.io/nick-myers-dt/pen/OJZbogL
Other browsers tested:
  Add OK or FAIL, along with the version, after other browsers where you
have tested this issue:
     Safari: OK
    Firefox: OK
       Edge: OK

What steps will reproduce the problem?
(1) Open https://codepen.io/nick-myers-dt/pen/OJZbogL
(2) Mirror Screen to Apple TV 
(3) Observe video is now black/blank

What is the expected result?
Video is displayed

What happens instead?
Video is not displayed (audio is played)
![Screenshot 2022-09-15 at 16 32 49](https://user-images.githubusercontent.com/57220787/190446494-bdb5a9b3-1a6d-4e5d-b78f-b27ed7952c4a.png)

**Comment by nick-myers-dt** [comment]
Issue on Apple Support community: https://discussions.apple.com/thread/254197199
Issue on Chromium bug reporter: https://bugs.chromium.org/p/chromium/issues/detail?id=1364125

**Comment by suzaku056** [comment]
> 

 Hi @evanluu, after we upgrade IOS to 16.1 have the same issue. Did you resolve this?
 Thank you!

**Comment by evanluu** [comment]
> Hi @evanluu, after we upgrade IOS to 16.1 have the same issue. Did you resolve this?
> Thank you!

Yes I can fix it by using npm not yarn nor pnpm. Just remove node_modules and package-lock.json and re-installing with npm.
